client/rpc/src/system/mod.rs

```diff
@@ -200,6 +200,11 @@ impl<B: traits::Block> SystemApi<B::Hash, <B::Header as HeaderT>::Number> for Sy
 
 	fn system_set_log_filter(&self, directives: String)-> std::result::Result<(), rpc::Error> {
 		self.deny_unsafe.check_if_safe()?;
-		sc_tracing::reload_filter(directives).map_err(|_e| rpc::Error::internal_error())
+		sc_tracing::reload_filter(directives, true).map_err(|_e| rpc::Error::internal_error())
+	}
+
+	fn system_raw_set_log_filter(&self, directives: String)-> std::result::Result<(), rpc::Error> {
```

Comment:
What is the plan here?

Reply:
the distinction between the two RPCs is documented in the API: https://github.com/paritytech/substrate/pull/7474/files#diff-74c2d14f1dd8ed14e698799a89d8768cdea798416794c0bc7255f1e5f28f8e09R111, basically this one excludes all default directives.